.github/workflows/build-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Build Website
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: cd docs-md
+    - run: npm ci
+    - run: npm install
+    - run: npm run check-links
+    - run: |
+        git config user.email "insights@fairwinds.com"
+        git config user.name "Circle CI"
```

Comment:
Since this is running in a GitHub workflow we probably don't want to use the name Circle CI. I wonder if for email address we should use `opensource@fairwinds.com`?

Reply:
Hah, copypasta. Fixed, thanks!